runtime/solochain-evm/src/configs/storage_hub.rs

```diff
@@ -57,3 +57,33 @@ impl Into<StorageEnableEvents<crate::Runtime>> for crate::RuntimeEvent {
         }
     }
 }
+
+// Implement transaction hash extraction for EVM runtime.
+impl shc_common::traits::TransactionHashProvider for crate::Runtime {
+    fn build_transaction_hash_map(
+        all_events: &shc_common::types::StorageHubEventsVec<Self>,
+    ) -> std::collections::HashMap<u32, sp_core::H256> {
+        let mut tx_map = std::collections::HashMap::new();
```

Comment:
I am surprised we are able to compile with `std` here in the runtime. Is there some obfuscated flag somewhere that I missed which only compiles it with the node/client binary, and not the runtime itself?